replate/src/components/Volunteer/Volunteer.js

```diff
@@ -0,0 +1,97 @@
+import React from "react";
+import { Router } from "react-router-dom";
+
+class Volunteer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      newVolunteer: {
+        firstName: "",
+        lastName: "",
+        phone: "",
+        email: "",
+        password: "",
+        repeatPassword: ""
+      }
+    };
+  }
+
+  changeHandler = e => {
+    this.setState({
+      newVolunteer: {
+        //Spreads previous state values, and changes current value depending on input from form
+        ...this.state.newVolunteer,
+        [e.target.name]: e.target.value
+      }
+    });
+  };
+
+  render() {
+    <div className="sign-up-wrapper">
+      <div className="sign-up-title">
+        <h2>Volunteer Sign Up</h2>
+        <p>Thanks for your interest!</p>
+      </div>
+      <form className="sign-up-form">
+        <h2>Create Your Account</h2>
+        <div className="input-field">
+          <p>First Name</p>
+          <input
+            type="text"
+            name="firstName"
+            onchange={this.changeHandler}
```

Comment:
Minor typo. `onchange` should be ``` onChange={ }```. Check all your `onChange` in this block of code.